packages/vsce/src/utils/resourceUtils.ts

```diff
@@ -37,13 +39,11 @@ export async function runGetResource({
     ...(params?.queryParams && { queryParams: params.queryParams }),
   };
 
-  logResourceRequest({
-    call: "runGetResource",
-    resourceName,
-    cicsPlex,
-    regionName,
-    ...params
-  });
+  CICSLogger.debug("GET request - Resource [" + resourceName + "]" +
```

Comment:
Wondering if specifying the parameters regardless of if they exist would be beneficial or not. 

Option A, `Resource [CICSProgram], Region [MYREG1]`
Option B, `Resource [CICSProgram], CICSPlex [undefined], Region [MYREG1]`

Technically option A shows that the plex name wasn't there, which would be why the request is failing, but option B shows that in a clearer way. 
On the other hand, when you're not wanting a plex name, it could be considered messy to have undefined's thrown around. But again, that's what you expect it to be so maybe that's okay?